### ISSUE TITLE
SparseCategoricalFocalLoss: ignore class?

### ISSUE THREAD

**Issue by rdisipio** [artemmavrin/focal-loss#24]
Hi,

   TensorFlow's [implementation](https://www.tensorflow.org/api_docs/python/tf/keras/losses/SparseCategoricalCrossentropy) of categorical cross-entropy accepts a parameter called `ignore_class` (see [here](https://github.com/keras-team/keras/blob/b80dd12da9c0bc3f569eca3455e77762cf2ee8ef/keras/backend.py#L5550)) since v 2.10. In principle, it can be useful for NER tasks. It would be nice if such a parameter appeared in the calculation of focal loss, so an apple-to-apple comparison can be carried out.
